tavern/_plugins/rest/request.py

```diff
@@ -111,6 +112,9 @@ def add_request_args(keys, optional):
     if "auth" in fspec:
         request_args["auth"] = tuple(fspec["auth"])
 
+    if "cert" in fspec:
+        request_args["cert"] = tuple(fspec["cert"])
```

Comment:
See comment below - this can be either a string or a tuple when given to requests so it should handle both eventualities here as well.